src/kent/static/style.css

```diff
@@ -5,3 +5,10 @@
 th, td {
     vertical-align: top;
 }
+
+pre code {
+    text-wrap: wrap;
+    text-wrap-mode: wrap;
+    overflow: scroll;
+    word-break: break-all;
+}
```

Comment:
This fixes the index page so it shows wrapped examples that have very very long lines.